src/components/Button/Button.module.css

```
@@ -72,10 +76,10 @@
 }
 
 /**
- * Loading icon rotiation style
+ * 1) Loading icon rotiation style
```

This is fine without the `1)` because it's describing the whole block.
```suggestion
 * Loading icon rotiation style
```